test/user.test.js

```diff
@@ -299,6 +299,39 @@ describe('User', function() {
       });
     });
 
+    it('skips token invalidation when the relation is not configured', (done) => {
+      // the test passed when the operation did not crash
+      const app = loopback({localRegistry: true, loadBuiltinModels: true});
+      app.set('remoting', {errorHandler: {debug: true, log: false}});
```

Comment:
We don't need to change `remoting` config because we are not making any REST calls.